website/docs/cartesian/cartesian-chart.md

```diff
@@ -54,12 +54,20 @@ An array of objects to be used as data points for the chart.
 
 A `string` value indicating the _key_ of each `data[number]` object to be used on the independent (x) axis for charting. E.g. `"day"` if you want to use the `"day"` field of the data points for the x-axis.
 
+:::info
+The `xKey` prop must point to a field that has either a `number` or `string` type, as these are the values that can be reasonably serialized by Reanimated.
```

Comment:
point or map? We might want to give a quick example here on what this means code-wise.